src/localisation/launch/ekf.launch

```diff
@@ -1,31 +1,5 @@
 <launch>
-
-    <!-- Start up the EKF -->
-    <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf" output="screen">
-        <param name="output_frame" value="odom_combined" />
-        <param name="base_footprint_frame" value="base_link" />
-        <param name="freq" value="60.0" />
-        <param name="sensor_timeout" value="1.0" />  
-        
-        <!-- The topics that input to the EKF -->
-        <!-- Description    | /topic    | message_type      -->
-
-        <!-- Source: ros_arduino_bridge (wheel encoders) -->
-        <!-- 2D Pose        | /odom     | nav_msgs/Odometry --> 
-        <param name="odom_used" value="true" />
-        <!-- Source: Razor 9DOF IMU -->
-        <!-- 3D Orientation | /imu_data | sensor_msgs/Imu   -->
-        <param name="imu_used" value="true" />
-        <!-- Source: GPS (translated to a Odometry message) -->
-        <!-- 2D Pose        | /gps      | nav_msgs/Odometry -->
-        <param name="gps_used" value="true" />
-        
-        <!-- 3D Pose        | /vo      | nav_msgs/Odometry -->
-        <param name="vo_used" value="false"/>
-
-        <remap from="gps" to="/gps_driver/odom" />
-        <remap from="imu_data" to="/imu" />
+    <node name="ekf" pkg="localisation" type="ekf" output="screen">
+        <param name="character" value="?" />
```

Comment:
Don't think we need this param anymore, looks like a carry-over from the sample package.